client/src/components/ScrollWithOverflow/index.js

```diff
@@ -93,11 +94,11 @@ export default class ScrollWithOverflow extends React.PureComponent<Props, State
   coverRef: { current: null | HTMLDivElement }
 
   eventNames = [
-    'mousedown', 'mousemove', 'mouseup', 'wheel',
-    'touchstart', 'touchmove', 'touchend'
+    'pointerdown', 'pointermove', 'pointerup',
```

Comment:
This change occurred in Three.js r120: https://github.com/mrdoob/three.js/wiki/Migration-Guide#r119--r120

Here's an example issue suggesting the fix that worked: https://github.com/mrdoob/three.js/issues/20294#issuecomment-689035894